roles/nf-core/tasks/methylseq_include.yml

```diff
@@ -0,0 +1,44 @@
+
+- name: create directory for {{ pipeline }}  resources
+  file:
+    path: "{{ ngi_containers }}/{{ pipeline }}"
+    state: directory
+
+# the eutils service will only tolerate 3 requests / second so make sure not to exceed this
+- name: download FASTA references
+  get_url:
```

Comment:
This module will automatically check if the file is already downloaded, right?

Reply:
yes, from the docs: 
```
if a checksum is passed to this parameter, and the file exist under the [dest](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html#ansible-collections-ansible-builtin-get-url-module-parameter-dest) location, the destination_checksum would be calculated, and if checksum equals destination_checksum, the file download would be skipped (unless [force](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html#ansible-collections-ansible-builtin-get-url-module-parameter-force) is true). If the checksum does not equal destination_checksum, the destination file is deleted.
```

Reply:
Awesome!